File: restapis/adminservice/adminservice/Models/Booking.cs
using System;
using System.Collections.Generic;

namespace adminservice.Models;

public partial class Booking
{
    public int BookingId { get; set; }

    public string StartLocation { get; set; } = null!;

    public string EndLocation { get; set; } = null!;

    public DateTime BookingDatetime { get; set; }

    public string BookingStatus { get; set; } = null!;

    public int UserId1 { get; set; }

    public int DriverId { get; set; }

    public float Km { get; set; }

    public virtual DriverInfo Driver { get; set; } = null!;

    public virtual ICollection<Feedback> Feedbacks { get; set; } = new List<Feedback>();

    public virtual User UserId1Navigation { get; set; } = null!;
}

File: restapis/driver-service/src/main/java/com/example/demo/entities/Booking.java
package com.example.demo.entities;
import java.time.LocalDate;
import java.time.LocalDateTime;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.EnumType;
import jakarta.persistence.Enumerated;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "booking")
@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class Booking {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "booking_id", nullable = false)
	private Integer bookingId;

	@Column(name = "start_location")
	private String startLocation;

	@Column(name = "end_location")
	private String endLocation;

	@Column(name = "booking_date_time")
	private LocalDateTime bookingDateTime;

	@Column(name = "start_date")
	private LocalDate startDate;

	@Column(name = "end_date")
	private LocalDate endDate;

	@Enumerated(EnumType.STRING)
	@Column(name = "booking_status")
	private BookingStatus bookingStatus;

	@ManyToOne
	@JoinColumn(name = "user_id1")
	private Users user;

	@ManyToOne
	@JoinColumn(name = "driver_id")
	private DriverInfo driver;

}

